cmd/ctr/commands/run/run_unix.go

```diff
@@ -155,6 +155,9 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 		if limit != 0 {
 			opts = append(opts, oci.WithMemoryLimit(limit))
 		}
+		for _, dev := range context.StringSlice("device") {
+			opts = append(opts, oci.WithLinuxDevice(dev, "rwm"))
```

Comment:
Consider make device permission as option ?

Reply:
it can be added in a follow up if it's important to ppl.  `ctr` is still a dev/testing tool and not a full featured cli